.github/workflows/quality.yml

```
@@ -12,94 +12,94 @@ permissions:
-          run: bun run test
+      - name: Test
+        run: bun run test
 
-        - name: Upload coverage reports to Codecov
-          if: ${{ !cancelled() }}
-          uses: codecov/codecov-action@v3
-          env:
-             CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+      - name: Upload coverage reports to Codecov
+        if: ${{ !cancelled() }}
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.4)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)